content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Spire SDS Integration
+linktitle: How integrate with Spire CA.
+description: Describes how to configure Istio to integrate with Spire CA via Secret Discovery Service.
+weight: 40
+keywords: [kubernetes,spiffe,spire]
+aliases:
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+[Spire](/docs/ops/integrations/spire/)
+
+[SPIRE](https://spiffe.io/docs/latest/spire-about/spire-concepts/) is a production-ready implementation of the SPIFFE APIs that performs node
+and workload attestation in order to securely issue SVIDs to workloads, and verify
+the SVIDs of other workloads, based on a predefined set of conditions. In an Istio deployment, SPIRE can be
+configured as a source of SVIDs for Istio workloads through integrating with [Envoy's SDS API](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret).

```

Comment:
```suggestion
[SPIRE](https://spiffe.io/docs/latest/spire-about/spire-concepts/) is a production-ready implementation of the SPIFFE specification that performs node and workload attestation in order to securely issue cryptographic identities to workloads running in heterogeneous environments.  
In an Istio deployment, SPIRE can be configured as a source of cryptographic identities for Istio workloads through the integration with [Envoy's SDS API](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret).
```

Reply:
`SVID` is not a common term in Istio, I'd suggest that we use `cryptographic identities` for clarity's sake.

Reply:
Great! I will add an explanation about the socket detection behavior and use cryptographic identities instead